scripts/bot-deployment/UpdateBotGCP.sh

```diff
@@ -1,17 +1,31 @@
 #!/bin/bash
 set -e
 
-if [ $# -ne 2 ]; then
-    echo "Incorrect number of arguments supplied! First argument is bot's name. Second argument is path to config file."
-    echo "example: ./UpdateBotGCP.sh ethbtc-monitor-bot ./ethbtc-monitor-bot-env.txt"
+if [ $# -eq 0 ]; then
+    echo "Incorrect number of arguments supplied! First argument is bot's name. Second optional argument is path to config file."
+    echo "example: ./UpdateBotGCP.sh ethbtc-monitor-bot [keep env variables]"
+    echo "example: ./UpdateBotGCP.sh ethbtc-monitor-bot ./ethbtc-monitor-bot-env.txt [Update env variables]"
     exit 1
 fi
 
-echo "Updating" $1
-echo "Using ENV file:" $2
-gcloud compute instances update-container $1 \
-    --container-image docker.io/umaprotocol/protocol:latest \
-    --container-env-file $2 \
-    --zone us-central1-a \
-    --container-restart-policy on-failure \
-    --container-stdin
+if [ $# -eq 1 ]; then
+    echo "Updating" $1
+    echo "Using no new enviroment variable. Keeping current configuration"
+    gcloud compute instances update-container $1 \
+        --container-image docker.io/umaprotocol/protocol:latest \
+        --zone us-central1-a \
+        --container-restart-policy on-failure \
+        --container-stdin
+
+fi
+
+if [ $# -eq 2 ]; then
```

Comment:
What happens if 3 inputs are provided?

Reply:
if you put in three then the configuration is wrong and nothing happens.

Reply:
Maybe we could add a quick `or` on the first if to also check against too many arguments.

https://unix.stackexchange.com/a/47585